build.js

```diff
@@ -0,0 +1,50 @@
+const StyleDictionary = require("style-dictionary").extend("config.json");
+
+StyleDictionary.registerFilter({
+  name: "filter-typography",
+  matcher: ({ attributes }) => {
+    return attributes.type === "typography";
+  },
+});
+
+StyleDictionary.extend({
```

Comment:
Extending the `config.json` file here trumps what's in `config.json`. I attempted splitting the platforms so the color config was still in `config.json`, but it ended up not transforming those tokens.